openvino_tensorflow/ie_basic_engine.cc

```diff
@@ -30,59 +31,96 @@ void IE_Basic_Engine::infer(
     std::vector<std::string>& param_names) {
   load_network();
   if (m_infer_reqs.empty()) {
-    m_infer_reqs.push_back(m_exe_network.CreateInferRequest());
+    m_infer_reqs.push_back(m_compiled_model.create_infer_request());
   }
 
   //  Prepare input blobs
-  auto func = m_network.getFunction();
-  auto parameters = func->get_parameters();
+  auto parameters = m_model->get_parameters();
   for (int i = 0; i < inputs.size(); i++) {
     if (inputs[i] != nullptr) {
 #if defined(OPENVINO_2021_2)
-      if (m_device != "MYRIAD" && m_device != "HDDL")
-        m_infer_reqs[0].SetBlob(input_names[i], inputs[i]->get_blob());
-      else {
-        auto input_blob = m_infer_reqs[0].GetBlob(input_names[i]);
-        MemoryBlob::Ptr minput = as<MemoryBlob>(input_blob);
-        auto minputHolder = minput->wmap();
-
-        auto inputBlobData = minputHolder.as<uint8_t*>();
-        size_t input_data_size = input_blob->byteSize();
-        inputs[i]->read((void*)inputBlobData, input_data_size);
+      if (m_device != "MYRIAD" && m_device != "HDDL") {
```

Comment:
Should this part be removed? as the code would no more be compatible with earlier OV versions